ArduCopter/mode_planckrtb.cpp

```diff
@@ -9,12 +9,14 @@ bool ModePlanckRTB::init(bool ignore_checks){
     //If we're ready to land, jump right to it
     if(copter.mode_planckland.init(ignore_checks)) {
       _is_landing = true;
+      _stored_yaw_mode = auto_yaw.mode();
```

Comment:
I think you could just call this once before the `init()` method. Storing the value does no harm if the init doesn't complete successfully.